setup.py

```diff
@@ -23,6 +23,8 @@
     CONNECTOR_DEPENDENCY,
     # snowpark directly depends on typing-extension, so we should not remove it even if connector also depends on it.
     "typing-extensions>=4.1.0, <5.0.0",
+    "protobuf>=5.28",
```

Comment:
fix it

Reply:
what is the fix? I.e., what version did we settle on?

Reply:
remove these two lines. No changes in setup.py